src/azure-cli-core/azure/cli/core/parser.py

```diff
@@ -404,15 +407,17 @@ def _check_value(self, action, value):  # pylint: disable=too-many-statements, t
                                                          "rerun automatically.".format(ext_name))
                                 self.exit(exit_code)
                             else:
-                                error_msg = 'Extension {} installed. Please rerun your command.'.format(ext_name)
+                                error_msg = "Extension {} installed. Please rerun your command. " \
```

Comment:
if installation default on, maybe continue run should default on also so that it's a complete experience for user. Otherwise, user has to interrupt anyway, no  matter it's install extension or run command again. what's your thinking?

Reply:
I agree.